client/state/jetpack-connect/selectors/is-remote-site-on-sites-list.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Internal dependencies
+ */
+import { getAuthorizationData, getJetpackSiteByUrl } from 'state/jetpack-connect/selectors';
```

Comment:
More potential circular references here.

Reply:
Thanks, fixed these too in in https://github.com/Automattic/wp-calypso/pull/39759/commits/00ce9e5dacf055df71c1d110f545491bca5839e3